src/Allegro.CosmosDb.Migrator/scripts/build.cmd

```diff
@@ -0,0 +1 @@
+dotnet build -c release
```

Comment:
All these scripts will not work when run from the /scripts subdirectory. How exactly should they be run? Maybe some description is needed in README?
On Mac OS or linux you can probably run them like this:
```
cd src/Allegro.CosmosDb.Migrator
./scripts/build.sh
```
I'm not sure about windows

Reply:
i'll delete them, they are useless